src/services/solana-client/SolanaClient.ts

```diff
@@ -222,6 +227,32 @@ class SolanaClient {
       url
     }
   }
+
+  _ipfs = (text: string): { type: SolanaNFTType; url: string } | null => {
+    // Look for 'https://ipfs.io/ipfs/<...alphanumeric...>' and that will be the metadata location
+    // e.g. https://ipfs.io/ipfs/QmWJC47JYuvxYw63cRq81bBNGFXPjhQH8nXg71W5JeRMrC
```

Comment:
this is fine but FYI

https://stackoverflow.com/questions/67176725/a-regex-json-schema-pattern-for-an-ipfs-cid